docker-compose.yml

```diff
@@ -54,7 +54,13 @@ services:
         published: 443
         protocol: udp
 
+# Mercure is installed as a Caddy module, prevent the Flex recipe from installing another service
+###> symfony/mercure-bundle ###
+###< symfony/mercure-bundle ###
+
 volumes:
   php_socket:
   caddy_data:
   caddy_config:
+###> symfony/mercure-bundle ###
+###< symfony/mercure-bundle ###
```

Comment:
hi @dunglas, is this duplicate placeholder expected?

Reply:
@rogeriolino Yes, we need the marker to indicate to flex that the configuration is already present in services & volumes section.